File: an2/MAP/laborator1/src/Repository/RepositoryNumarComplex.java
package Repository;

import Domain.NumarComplex;
import Domain.Operation;

import java.util.ArrayList;

public class RepositoryNumarComplex {
    private final ArrayList<NumarComplex> numereComplexe;
    private final ArrayList<Operation> operations;

    public RepositoryNumarComplex(){
        this.operations = new ArrayList<>();
        this.numereComplexe = new ArrayList<>();
    }

    public void createNumarComplex(double real, double imaginar){
        NumarComplex numarComplex = new NumarComplex(real, imaginar);
        numereComplexe.add(numarComplex);
    }

    public void createOperator(Operation operation){
        operations.add(operation);
    }

    public ArrayList<NumarComplex> getNumereComplexe(){
        return numereComplexe;
    }

    public ArrayList<Operation> getOperations() {
        return operations;
    }
}

File: an2/MAP/lab9/lab1cs/Repository/RepositoryNumarComplex.cs
using lab9.Domain;

namespace Repository
{
    public class RepositoryNumarComplex
    {
        private List<NumarComplex> NumereComplexe;
        private List<Operation> Operations;

        public RepositoryNumarComplex()
        {
            this.Operations = new List<Operation>();
            this.NumereComplexe = new List<NumarComplex>();
        }

        public void CreateNumarComplex(double real, double imaginar)
        {
            NumarComplex numarComplex = new NumarComplex(real, imaginar);
            NumereComplexe.Add(numarComplex);
        }

        public void CreateOperator(Operation operation)
        {
            Operations.Add(operation);
        }

        public List<NumarComplex> GetNumereComplexe()
        {
            return NumereComplexe;
        }

        public List<Operation> GetOperations()
        {
            return Operations;
        }
    }
}